### ISSUE TITLE
monospaced

### ISSUE THREAD

**Issue by boorishpea** [googlei18n/noto-fonts#638]
Are there any plans to generate monospaced fonts?

**Comment by dougfelt** [comment]
Phase 3 plans to include a font that covers most of LGC and some box drawing and block characters.  No other monospace font is planned.

**Comment by TidharC** [comment]
Can this issue be closed?

**Comment by dougfelt** [comment]
I don't think we have a roadmap document, perhaps Xiangye knows.  I think we should have one, if xiangye says we already have one, or thinks we shouldn't perhaps she can close this.

**Comment by xiangyexiao** [comment]
Actually, we already alpha released monospace font for Latin Greek and Cyrillic. Find the source here:
https://github.com/googlei18n/noto-source/blob/master/src/NotoSansUIMono-MM.glyphs

You can run fontmake to build TTF/OTF fonts using the source.
https://github.com/googlei18n/fontmake

**Comment by dougfelt** [comment]
Yes, but I think we need to add more characters to monospace.

**Comment by xiangyexiao** [comment]
@dougfelt what characters are you considering? Currently, Monospace has the same character coverage as Latin-Greek-Cyrillic font. Are you thinking about more symbol characters?

**Comment by dougfelt** [comment]
Yes.  In particular, there are symbols used with programming languages
(APL, Haskell), and terminal graphics characters like block characters.

**Comment by dougfelt** [comment]
​In particular, search for 'Mono' issues under noto-fonts and you'll find a
number of requests for additional Monospace characters, as well as a
request for italic monospace.

**Comment by codeman38** [comment]
What is the best place to file a bug regarding the alpha monospaced fonts? I've found at least one showstopper issue in them: the "@" character is double-wide.

**Comment by dougfelt** [comment]
I think noto-fonts is the place.
